OpenRA.Mods.Common/Lint/CheckSequences.cs

```diff
@@ -26,14 +26,13 @@ class CheckSequences : ILintMapPass
 
 		public void Run(Action<string> emitError, Action<string> emitWarning, Map map)
 		{
-			if (map != null && !map.SequenceDefinitions.Any())
+			if (map != null || map.SequenceDefinitions == null)
```

Comment:
`map == null || map.SequenceDefinitions == null`

Caught an NRE before Coverity for once! \o/